examples/with-material-ui/src/App.js

```diff
@@ -1,31 +1,30 @@
 import './App.css';
 import React from 'react';
 import PropTypes from 'prop-types';
-import { withStyles } from 'material-ui/styles';
-import Paper from 'material-ui/Paper';
-import Typography from 'material-ui/Typography';
+import Paper from '@material-ui/core/Paper';
```

Comment:
Since Material-UI now does tree-shaking, these imports can be safely changed to `import { Paper, Typography } from '@material-ui/core'`. That being said, webpack >= 2.0 or parcel with a certain flag is needed. https://material-ui.com/guides/minimizing-bundle-size/#how-to-reduce-the-bundle-size

Reply:
That's the point, this would break razzle as mentioned in this issue: https://github.com/jaredpalmer/razzle/issues/1035 
And here the linked change: https://github.com/vuhrmeister/razzle/compare/fix-material-ui-example...material-ui-broken-build

Reply:
Ah, interesting. I'll have to take a look at that one, because I haven't come across that issue in any of my environments yet. But I agree, leave it as is!